### PR TITLE
Document the design of the A4 data model

### PR DESCRIPTION
**Pull request by slomo**

### CODE REVIEW

File: docs/datamodel.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+`a4modules` and `a4phases` provide the basic infrastructure to model
+participation processes. In order to write a well adapted participation
+software, additional apps can be created that base on the functionality on the 
+core and provide more features.
+
+participation process model
+---------------------------
+
+The adhocracy4 process model always replicates the follwoing data model: A set
+of organisations on the platform runs each their own participation projects.
```
Comment (xi) [inline docs/datamodel.md]: "Organisation" is not a core concept, is it?

Comment (2e2a) [inline docs/datamodel.md]: It is, as it is required by a project for the initiator permissions.

**Comment by MagdaN** [comment]
@xi @2e2a this looks good to me. Should we merge?

**Comment by xi** [comment]
I still would like to improve some parts, but that can be done. later. Merging.

**Pull request merged**